src/bundle/Resources/public/js/scripts/core/draggable.js

```diff
@@ -128,7 +129,17 @@
 
             const positionY = this.getPlaceholderPositionTop(item, event);
 
+            if (item.isSameNode(this.placeholderPositionData.item) && this.placeholderPositionData.positionY === positionY) {
```

Comment:
Just small optimization change - if mouse doesn't move while dragging, it now doesn't rerender placeholder (dragging event is called each few milliseconds, not on mousemove)